src/generators/static/tsConfigFileGenerator.ts

```diff
@@ -46,16 +46,23 @@ const restLevelTsConfig = {
 };
 
 export function generateTsConfig(project: Project) {
-  const { generateMetadata, restLevelClient } = getAutorestOptions();
+  const {
+    generateMetadata,
+    restLevelClient,
+    skipLibCheck
+  } = getAutorestOptions();
 
   if (!generateMetadata) {
     return;
   }
 
-  const tsConfigContents = restLevelClient
+  const tsConfigContents: any = restLevelClient
```

Comment:
I would rather not do this and instead define an interface for things we support in tsconfig.